### PR TITLE
Constants in code should be replaced with MACRO

### PR DESCRIPTION
**Pull request by amitkumar50**
replacing constant value 1 in ceph/src/os with macro VAL1

Resolves: http://tracker.ceph.com/issues/20698

Signed-off-by: Amit Kumar <amitkuma@redhat.com>

### CODE REVIEW

**Comment by amitkumar50** [comment]
@joscollin can you review please!!!

File: src/common/config.h [hunk 1]
```diff
@@ -25,6 +25,7 @@
 #define OSD_REP_SPLAY   1
 #define OSD_REP_CHAIN   2
 
+#define VAL1 1
```
Comment (tchaikov) [inline src/common/config.h]: i don't think this helps with readability.

Comment (joscollin) [inline src/common/config.h]: @tchaikov Thanks for the suggestion. I wanted someone else to review this cleanup.

Comment (amitkumar50) [inline src/common/config.h]: Variable name can be changed!!! your suggestions would be good..
But using constant in code ... I believe is no Good..

Comment (amitkumar50) [inline src/common/config.h]: > please be specific. in the case of your PR of https://github.com/ceph/ceph/pull/16447, replacing 1 with VAL1 does not help at all. if 1 is a magic number, for example, which is a sentry of a number sequence. we need to have a better name for it. but in this case, it's not.
> 

@tchaikov I think `#define ONE 1` would be good, Do you have any other meaningful name that is appropriate to the module.

**Pull request closed without merging**